[RHACS] ROX-22231: cloud source integrations (#71977)
openshift/openshift-docs: modules/cloud-source-paladin-cloud.adoc

// Module included in the following assemblies:
//

> // * integration/integrate-with-paladin-cloud.adoc

Author: It should be `integrate-with-cloud-sources.adoc`.

:_mod-docs-content-type: PROCEDURE
[id="cloud-source-paladin-cloud_{context}"]
= Configuring Paladin Cloud integration

To discover cluster assets from Paladin Cloud, create a new integration in {product-title}.

.Prerequisites
* A Paladin Cloud account.
* A Paladin Cloud API token.

.Procedure
. On the {product-title-short} portal, navigate to *Platform Configuration* -> *Integrations*.
. Scroll down to the *Cloud source integrations* section and select *Paladin Cloud*.

> . Click *New Integration* (*`add`* icon).

Reviewer: I'm not super familiar with UX terminology, so my feedback is limited here, but I think unless "add" appears as a phrase in a file or code output, basically anything you'd see on a CLI, the literals shouldn't be used here:
```suggestion
. Click *New Integration* (*add* icon).
```

[RHSSG link](https://redhat-documentation.github.io/supplementary-style-guide/#user-interface-elements) for reference.

Author: I copied this from https://github.com/openshift/openshift-docs/blob/rhacs-docs/modules/amazon-s3-configuring-acs.adoc. Tbh I'm not sure what the purpose is, the button is called `New integration` and has no icon.

Author: I removed the entire "(*`add`* icon)" part.

. Enter a name for *Integration name*.

> . Enter the Paladin Cloud API endpoint for *Paldin Cloud endpoint*. Defaults to https://api.paladincloud.io.

Reviewer 2: One thing I'm a bit worried about at the moment: this API endpoint doesn't exist (alas, the domain name itself doesn't resolve yet with NXDOMAIN); worthwhile to check in with folks whether we should really advertise this as the API endpoint or be generic and say "your Paladin endpoint" (whatever this would be).

Author: I write another email to Paul to ask when this API will be available. Let's keep the default for now.

> . Enter the Paladin Cloud API endpoint for *Paldin Cloud endpoint*. Defaults to https://api.paladincloud.io.

Reviewer: Second sentence should be rewritten into a complete sentence:
```suggestion
. Enter the Paladin Cloud API endpoint for *Paldin Cloud endpoint*. The default is https://api.paladincloud.io.
```

> . Enter the Paladin Cloud API endpoint for *Paldin Cloud endpoint*. Defaults to https://api.paladincloud.io.

Reviewer: I believe this is a spelling error:
```suggestion
. Enter the Paladin Cloud API endpoint for *Paladin Cloud endpoint*. Defaults to https://api.paladincloud.io.
```

. Enter the Paladin Cloud API token for *Paldin Cloud token*.

> . Select *Test* (*`checkmark`* icon) to confirm that authentication is working.

Author: see above, removed the bracket part.

> . Select *Create* (*`save`* icon) to create the configuration.

Author: see above, removed the bracket part.
